app/src/main/java/sdp/moneyrun/PlayerProfileActivity.java

```diff
@@ -31,6 +31,7 @@ protected void onCreate(Bundle savedInstanceState) {
             public void onClick(View v) {
                 Intent mainMenuIntent = new Intent(PlayerProfileActivity.this, MenuActivity.class);
                 startActivity(mainMenuIntent);
+                finish();
```

Comment:
Yeah, I noticed that it was never really closed and sometimes when coming back from the map it would lead to the profile activity and not the menu, but this fixed it.